zwave_js/build.json

```diff
@@ -7,7 +7,7 @@
     "aarch64": "homeassistant/aarch64-base:3.13"
   },
   "args": {
-    "ZWAVEJS_SERVER_VERSION": "1.1.0",
+    "ZWAVEJS_SERVER_VERSION": "1.1.1",
     "ZWAVEJS_VERSION": "6.5.0"
```

Comment:
Should we bump zwave-js too? It that safe?

https://github.com/zwave-js/node-zwave-js/releases

Reply:
The one change that's "breaking" is the switch to "buffer" for the value type if the result is returned as a buffer object. We would need to add support for that in the library, otherwise those values won't be stored.

Reply:
If we are ok with that limitation then I can bump zwave-js as well

Reply:
Do we support the values today that would become buffer type?

Reply:
We do: https://github.com/home-assistant-libs/zwave-js-server-python/blob/master/zwave_js_server/model/value.py#L236

The change is to switch the type here from `string` to `buffer` for these scenarios

Reply:
Ok, then we should add support first in the library and Home Assistant before we bump.